docs/blog/posts/2021-10-06-sharedTest.md

```diff
@@ -5,6 +5,8 @@ authors:
 slug: sharedTest
 ---
 
+<!-- markdownlint-disable MD052 -->
```

Comment:
Numbered reference links don't seem to be recognized by the linter. I'll update them in a separate PR to not block this one any longer.

https://github.com/DavidAnson/markdownlint/blob/v0.36.0/doc/md052.md